app/models/user.rb

```diff
@@ -7,4 +7,5 @@ class User < ApplicationRecord
   has_many :garden_plants
   has_many :plants, through: :garden_plants, dependent: :destroy
   has_many :comments, dependent: :destroy
+  validates :email, presence: true, format: URI::MailTo::EMAIL_REGEXP
```

Comment:
I included this flag to guard against invalid email formats, I will look more into this regex to see if a better option is available as this can also be bypassed with imagination.